src/Nethermind/Nethermind.Consensus/Validators/BlockValidator.cs

```diff
@@ -405,7 +405,7 @@ private bool ValidateEip4844Fields(Block block, IReleaseSpec spec, out string? e
 
         ulong blobGasUsed = BlobGasCalculator.CalculateBlobGas(blobsInBlock);
 
-        if (blobGasUsed > Eip4844Constants.MaxBlobGasPerBlock)
+        if (!spec.IsEip7742Enabled && blobGasUsed > Eip4844Constants.MaxBlobGasPerBlock)
```

Comment:
but if EIP enabled than we do what? I think it is not handled correctly

Reply:
> Upon activating this EIP (i.e. before processing any transactions), the verification of the blob maximum as given in EIP-4844 can be skipped. Concretely, this means any logic relating to MAX_BLOB_GAS_PER_BLOCK as given in EIP-4844 can be deprecated
This is written in EIP specification, I think if `IsEip7742Enabled = true` we need to skip this logic

Reply:
i think its assumed that the CL will do the verification regarding the MaxBlockGas/MaxBlobCount because that value is provided for blockProduction?